### PR TITLE
Display request/response time

### PR DESCRIPTION
**Pull request by paulstatezny**
Would be handy to know how many ms each API request takes so that slow ones can be inspected more closely.

### CODE REVIEW

**Comment by baohx2000** [comment]
@paulstatezny I have a local branch merged to re-do bits of this which were wiped out from a merge of master, but the results aren't matching up with devtools.  Will talk to you about it in the morning.

**Comment by baohx2000** [comment]
:m: :star: note that time given includes the OPTIONS request...nothing we can do about that at this time without rewriting the lower level request stuff ourselves

File: application/ui/components/endpoint.jsx [hunk 1]
```diff
@@ -299,10 +300,13 @@ module.exports = React.createClass({
 
         if (this.state.status === LOADED)
         {
+            var latency = this.state.responseTimestamp > 0 ? (parseInt(this.state.responseTimestamp) - parseInt(this.state.requestTimestamp)) : 0;
             apiCallInfo = (
                 <ApiCallInfo status={this.state.status}
                              request={this.state.requestInfo}
-                             response={this.state.response} />
+                             response={this.state.response}
+                             latency={latency}
```
Comment (paulstatezny) [inline application/ui/components/endpoint.jsx]: We try to align these, with a space on either side of the `=`. Looks like that wasn't done here originally.

File: application/store/request.js [hunk 2]
```diff
@@ -160,7 +162,9 @@ module.exports = Fluxxor.createStore({
         return {
             values         : [],
             excludedFields : [],
-            nullFields     : []
+            nullFields     : [],
+            requestTimestamp: 0,
```
Comment (agerbens) [inline application/store/request.js]: Alignment

**Comment by paulstatezny** [comment]
:pray:

**Comment by dmmcinty** [comment]
:hand:

**Comment by dmmcinty** [comment]
:+1:

**Pull request merged**